apps/framework-cli/src/cli/local_webserver.rs

```diff
@@ -337,13 +363,16 @@ async fn ingest_route(
     }
 }
 
+#[allow(clippy::too_many_arguments)]
```

Comment:
I still think making the `project` a global (immutable) variable is a good idea.

Reply:
Hum - This method doesn't hava `Project` in it - not sure that makes the point